frontend/src/app/main-dashboard/main-dashboard.component.ts

```diff
@@ -0,0 +1,47 @@
+/* Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. */
+
+import { Component } from '@angular/core';
+import { map } from 'rxjs/operators';
+import { Breakpoints, BreakpointObserver } from '@angular/cdk/layout';
+
+@Component({
+  selector: 'app-main-dashboard',
+  templateUrl: './main-dashboard.component.html',
+  styleUrls: ['./main-dashboard.component.css']
+})
+export class MainDashboardComponent {
+  /** Based on the screen size, switch from standard to one column per row */
+
+  cards = this.breakpointObserver.observe(Breakpoints.Handset).pipe(
+    map(({ matches }) => {
+      if (matches) {
+        // Phone version.
+        return [
+          { title: 'Card 1', cols: 1, rows: 1 },
+          { title: 'Card 2', cols: 1, rows: 1 },
+          { title: 'Card 3', cols: 1, rows: 1 }
```

Comment:
Is this a dummy placeholder for now?

Reply:
This renders if a phone opens the app. Mobile use isn't really a use case so I'll remove that. This template was autogenerated by importing the MainDashboard schematic from Angular, I just filled in the title with 'Legend' and 'Options'